Rename keb.components to keb.component (#164)
kyma-incubator/reconciler: README.md

./bin/reconciler-darwin local --components tracing,monitoring --value tracing.key=value,global.key=value
```

## Testing

### Unit tests

> Use the `make test` target to execute unit tests:

Reviewer: ```suggestion
To execute unit tests, use the `make test` target:
```

      make test

### Integration test

> Integration tests have a higher resource consumption compared to unit tests and expect that the environment variable `KUBECONFIG` is defined which points to a test Kubernetes cluster. 

Reviewer: ```suggestion
Integration tests have a higher resource consumption compared to unit tests. You must define the environment variable `KUBECONFIG`, which points to a test Kubernetes cluster. 
```

> Be aware that the integration test suite will install and delete Kubernetes resources as part of the test run.

Reviewer: ```suggestion
Be aware that the integration test suite installs and deletes Kubernetes resources during the test run.
```

> To execute the integration tests please execute the `make test-all` target:

Reviewer: ```suggestion
To execute the integration tests, execute the `make test-all` target:
```
